src/tests/ftest/server/metadata.py

```diff
@@ -199,48 +256,28 @@ def test_metadata_fillup(self):
         :avocado: tags=metadata_fillup,test_metadata_fillup
         """
         self.create_pool()
+        svc_ops_entry_age = self.pool.get_property("svc_ops_entry_age")
+        if not self.run_dummy_metadata_workload(duration=svc_ops_entry_age):
+            self.fail("failed to run dummy metadata workload")
 
-        # 3 Phases in nested try/except blocks below
-        # Phase 1: overload pool metadata with a container create loop
-        #          DaosApiError expected here (otherwise fail test)
-        #
-        # Phase 2: if Phase 1 passed:
-        #          clean up all containers created (prove "critical" destroy
-        #          in rdb (and vos) works without cascading no space errors
-        #
-        # Phase 3: if Phase 2 passed:
-        #          Sustained container create loop, eventually encountering
+        # 2 Phases
+        # Phase 1: Sustained container create loop, eventually encountering
         #          -DER_NOSPACE, perhaps some successful creates (due to
         #          rdb log compaction, eventually settling into continuous
         #          -DER_NOSPACE. Make sure service keeps running.
         #
+        # Phase 2: if Phase 1 passed:
+        #          clean up several (not all) containers created (prove "critical" destroy
+        #          in rdb (and vos) works without cascading no space errors
 
-        # Phase 1 container creates
-        self.log.info("Phase 1: Fill up Metadata (expected to fail) ...")
-        if not self.create_all_containers():
-            self.fail("Phase 1: failed (metadata full error did not occur)")
-        self.log.info(
-            "Phase 1: passed (container create %d failed after metadata full)",
-            len(self.container) + 1)
-
-        # Phase 2 clean up containers (expected to succeed)
-        self.log.info(
-            "Phase 2: Cleaning up %d containers (expected to work)", len(self.container))
-        if not self.destroy_all_containers():
-            self.fail("Phase 2: fail (unexpected container destroy error)")
-        self.log.info("Phase 2: passed")
-
-        # Phase 3 sustained container creates even after no space error
-        # Due to rdb log compaction after initial no space errors, some brief
-        # periods of available space will occur, allowing a few container
-        # creates to succeed in the interim.
+        # Phase 1 sustained container creates even after no space error
         self.log.info(
-            "Phase 3: sustained container creates: to no space and beyond")
+            "Phase 1: sustained container creates: to no space and beyond")
         self.container = []
         sequential_fail_counter = 0
-        sequential_fail_max = 1000
+        sequential_fail_max = 512
```

Comment:
Please add sequential_fail_max var to metadata.yaml.

Reply:
added this in the latest code on the branch.